build/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -eo pipefail
```

Comment:
We don't need to set these in the scripts. We're setting them in the execution instead...